ExpressFirstApp/index.js

```diff
@@ -16,6 +18,68 @@ app.use(express.static(staticDirectory));
 app.use(accessLogger);
 app.use(durationLogger);
 app.use(corsMiddleware);
+app.use(bodyParser.json());
+
+//Creating new todo files or appending to already created
+app.post('/todo', (req, res) => {
+  const { todoName, id, task } = req.body;
+  const log = `${id}. ${task} \n`;
+
+  fs.appendFile(`./todo/${todoName}.txt`, log, (err) => {
+    if (err) {
+      throw err;
+    }
+  });
+});
+
+//Getting the content from to do lists
+app.get('/todo', (req, res) => {
+  const { todoName } = req.body;
+
+  fs.readFile(`./todo/${todoName}.txt`, 'utf-8', (err, data) => {
+    if (err) {
+      console.error(err.message);
+    } else {
+      console.log(`${todoName}: `);
+      console.log(data);
```

Comment:
I can't find where the content is responded to the client. Is it only logged to the console?